scrapi/migrations.py

```diff
@@ -100,6 +151,7 @@ def next_page_source_partition(query, page):
 
 documents_old = ModelIteratorFactory(DocumentModelOld, next_page_old)
 documents = ModelIteratorFactory(DocumentModel, next_page_source_partition, default_args=registry.keys())
+postgres_documents = Document.objects.all()
```

Comment:
I would make it so that each processor exposes an iterator, rather than adding more of these